apps/frontend/src/components/ComponentsList/ComponentsList.tsx

```diff
@@ -4,6 +4,7 @@ import Draggable from '@components/Decorators/Draggable';
 import ComponentWrapper from '@components/ComponentWrapper/ComponentWrapper';
 
 interface ComponentsListProps {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
I think we can find a better typing solution for this, instead of disabling `eslint` each time.

Let's keep it in mind.